### PR TITLE
Created metrics for range answers

### PR DESCRIPTION
**Pull request by Thomas-Leung**

### CODE REVIEW

File: src/main/java/com/studymonkey/surveychimp/viewControllers/AnswerController.java [hunk 1]
```diff
@@ -88,6 +86,21 @@ else if (q.getQuestionType() == QuestionType.RANGE) {
...
+    public List<Answer> getQuestionAnswersRangeBarchart(@PathVariable long questionId, Model model) {
+        Question q = this.questionRepository.findById(questionId);
+
+        if (q == null) {
+            System.out.println("Question does not exist");
+            return null;
+        }
+
+        return q.getAnswers();
+    }
```
Comment (ChristopheTran) [inline src/main/java/com/studymonkey/surveychimp/viewControllers/AnswerController.java:104]: Hey I think theres a method called getQuestionAnswersJSON() that does the exact same thing.
We can use this one if you prefer :P

**Review by ChristopheTran — COMMENTED** [review]
Overall looks good!

I'll just wait for you to address my comment and I'll approve.

File: src/main/resources/templates/rangeanswers.html [hunk 2]
```diff
@@ -48,7 +52,14 @@
     </div>
 </nav>
 
-<h2 th:text="${question.question} + ' (' + ${question.min} + '-' + ${question.max} + ')'"></h2>
+<div style="padding: 8px">
+    <h2 id="questionTitle" th:text="${question.question} + ' (' + ${question.min} + '-' + ${question.max} + ')'"></h2>
+    <!--<a th:field="*{questionId}" th:href="@{/answer/rangeAnswer/barchart/{questionId}(questionId=${question.id})}" class="btn btn-primary">See metrics</a>-->
```
Comment (MichaelPatsula) [inline src/main/resources/templates/rangeanswers.html:57]: Can you please remove the comment?

Comment (Thomas-Leung) [reply src/main/resources/templates/rangeanswers.html:57]: What do you mean the title of the HTML is still Text Answer?

File: src/main/resources/templates/rangequestioncreate.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="about">About</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<div class="container">
```
[the remark this answers is not in the document]
Comment (Thomas-Leung) [reply src/main/resources/templates/rangequestioncreate.html:47]: This is some artifacts before you changed the backend. Will be removed.

File: src/main/resources/templates/rangeanswers.html [hunk 4]
```diff
@@ -9,6 +9,10 @@
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
             integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"
             crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+    <!-- Because the path is at answer/questionAnswer in Spring for this html, so we need ../../-->
+    <script src="../../rangebarchart.js"></script>
```
[the remark this answers is not in the document]
Comment (Thomas-Leung) [reply src/main/resources/templates/rangeanswers.html:15]: Spring will think the rangebarchart.js is located in answer/questionAnswer, however, our js file is located at root. Therefore, we need ../../.

**Comment by Christopher-Wang** [comment]
I think this is ready to be merged

**Review by RahulAnilkumar8 — APPROVED** [review]
Overall I think its good to go. Merge when you are ready.

**Pull request merged**